src/Navbar.js

```diff
@@ -19,256 +21,408 @@ import MenuIcon from '@mui/icons-material/Menu';
 import ExpandMoreIcon from '@mui/icons-material/ExpandMore';
 import ArrowUpwardIcon from '@mui/icons-material/ArrowUpward';
 import ArrowDownwardIcon from '@mui/icons-material/ArrowDownward';
-import './Navbar.css';
 
 export default function Navbar(props) {
+    const isNarrowThan300 = useMediaQuery({ maxWidth: 300 })
```

Comment:
I'd rename this to 'isNarrow'. No need to hardcode the width in the variable name.
If you want to keep it, make it grammatical: isNarrowerThan300